scripts/ansible/ansible.cfg

```diff
@@ -1,6 +1,16 @@
 [defaults]
+any_errors_fatal = true
 transport = ssh
 deprecation_warnings = false
+host_key_checking = false
 
 [ssh_connection]
-ssh_args = -o ForwardAgent=yes -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null
+# Optimisation, speeds up playbook execution by reducing network round trips.
+# However, remote systems must not have requiretty set in /etc/sudoers.
+pipelining = true
+ssh_args =
+    -C
+    -o ControlMaster=auto
+    -o ControlPersist=60s
+    -o ForwardAgent=yes
+    -o UserKnownHostsFile=/dev/null
```

Comment:
I am not sure about this entry either at the global level?

Reply:
Again, I haven't added `UserKnownHostsFile=/dev/null`, I just happened to reformat that line.

Perhaps your concern is that this ansible.cfg has any effect outside this directory? It is only in effect when this directory is the working directory.